cli/spec/kontena/command_spec.rb

```diff
@@ -0,0 +1,55 @@
+require_relative '../spec_helper'
+require 'kontena/command'
+
+describe Kontena do
+  let(:subject) do
+    class TestCommand < Kontena::Command
```

Comment:
Use `Class.new(Kontena::Command) do ... ` for spec classes, because the `TestCommand` pollutes the global ruby namespace.

IIRC this will fail if you have two specs that both try and use `class TestCommand`.